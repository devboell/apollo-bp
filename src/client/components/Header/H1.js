import styled from 'styled-components'

const H1 = styled.h1`
  margin: 5px 0px 0px 20px;
  flex-basis: 80%;
`

export default H1
