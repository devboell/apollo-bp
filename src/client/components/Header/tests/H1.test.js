import React from 'react'
import 'jest-styled-components'

import H1 from '../H1'

it('H1, styled component snapshot', () => {
  const wrapper = shallow(<H1 />)
  expect(wrapper).toMatchStyledComponentsSnapshot()
})
