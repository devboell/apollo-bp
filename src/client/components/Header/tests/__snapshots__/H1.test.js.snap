// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1, styled component snapshot 1`] = `
.hGdliM {
  margin: 5px 0px 0px 50px;
}

<h1
  className="sc-bdVaJa hGdliM"
/>
`;
